hardware_interface:
    joint_offsets: # degrees
        j1: 0.0
        j2: 0.0
        j3: 0.0
        j4: 0.0
        j5: 0.0
        j6: 0.0
